src/components/XkcdComicCardHomePage.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import {XkcdComicCard} from "./XkcdComicCard";
+
+export const Content = () => {
```

Comment:
This probably should also take the props and pass them to the Card:

```ts
import {XkcdComicProps} from "../types";
export const Content = (props: XkcdComicProps) => {
  return (<XkcdComicCard {...props}/>);
};

```

Reply:
Good catch. Done.